bcipy/parameters/parameters.json

```diff
@@ -1,749 +1,749 @@
-    "recommended_values": "",
-    "type": "directorypath"
-  },
-  "lm_backspace_prob": {
-    "value": "0.05",
-    "section": "bci_config",
-    "readableName": "Backspace Probability",
-    "helpTip": "Specifies the initial probability assigned to the backspace character in the language model. Possible value range: 0.0-1.0. Default: 0.05",
-    "recommended_values": "",
-    "type": "float"
-  },
-  "show_preview_inquiry": {
-    "value": "false",
-    "section": "bci_config",
-    "readableName": "Preview Inquiry On/Off",
-    "helpTip": "If ‘true’, the inquiry will be previewed as applicable for the task. *Note* Not all tasks will have this enabled!",
-    "recommended_values": "",
-    "type": "bool"
-  },
-  "preview_inquiry_progress_method": {
-    "value": "0",
-    "section": "bci_config",
-    "readableName": "Preview Inquiry Progression Method",
-    "helpTip": "If show_preview_inquiry true, this will determine how to proceed after a key hit. 0 = preview only; 1 = press to confirm; 2 = press to skip to another inquiry",
-    "recommended_values": [
-      "0",
-      "1",
-      "2"
-    ],
-    "type": "int"
-  },
-  "preview_inquiry_length": {
-    "value": "5",
-    "section": "bci_config",
-    "readableName": "Preview Inquiry Display Length",
-    "helpTip": "Length of time in seconds to present an inquiry preview to the user.",
-    "recommended_values": "",
-    "type": "float"
-  },
-  "preview_inquiry_key_input": {
-    "value": "return",
-    "section": "bci_config",
-    "readableName": "Preview Inquiry Display Key Input Method",
-    "helpTip": "Defines the key used to engage with inquiry preview.",
-    "recommended_values": [
-      "space",
-      "escape",
-      "return"
-    ],
-    "type": "str"
-  },
-  "preview_inquiry_isi": {
-    "value": "1",
-    "section": "bci_config",
-    "readableName": "Preview Inquiry Inter-Stimulus Interval",
-    "helpTip": "The time between previewing an inquiry and the start of an inquiry.",
-    "recommended_values": "",
-    "type": "float"
-  },
-  "show_feedback": {
-    "value": "true",
-    "section": "bci_config",
-    "readableName": "Feedback On/Off",
-    "helpTip": "If ‘true’, feedback will be shown after each inquiry.",
-    "recommended_values": "",
-    "type": "bool"
-  },
-  "feedback_duration": {
-    "value": "2",
-    "section": "bci_config",
-    "readableName": "Feedback Time (seconds)",
-    "helpTip": "Specifies the length in time (seconds) feedback will be displayed after each inquiry in registered tasks (ex. RSVP Copy Phrase). Default: 2",
-    "recommended_values": "",
-    "type": "float"
-  },
-  "psd_method": {
-    "value": "Welch",
-    "section": "bci_config",
-    "readableName": "Power Spectral Density Method",
-    "helpTip": "Specifies the method used to approximate power spectral density bands (Welch or MultiTaper). Default: Welch",
-    "recommended_values": [
-      "Welch",
-      "MutliTaper"
-    ],
-    "type": "str"
-  }
+{

```

Comment:
I'm not sure why this file shows such a large diff. Did anything actually change?

Reply:
I updated the alert tone path. I'm not sure why it looks so changed? I'll try resetting it.